framework/kms-keyring.md

```diff
@@ -73,6 +73,9 @@ and MAY return a KMS Client that can make the following API calls in the given A
 
 The keyring will use this client supplier to determine the KMS client to use when making KMS calls.
 
+The purpose of the client supplier is
```

Comment:
What about something like:

To communicate with AWS KMS the AWS KMS Keyring needs a client.
The client suppliers purpose is to configure that client.
An example might be different timeouts per region.
It is not intended to provide control flow or filtering.

I also wonder if we should lead with this?